### PR TITLE
Make navigation scrollable

### PR DESCRIPTION
**Pull request by dmitrydao**

### CODE REVIEW

**Review by StephenFluin — DISMISSED** [review]
This seems to break the ability to resize the page. Now the menu is collapsing prematurely. See screenshot.

localhost_4200_ (1)

File: _src/docs.css [hunk 1]
```diff
@@ -121,9 +121,14 @@ a:hover,
 }
 navigation {
```
Comment (alexroan) [inline _src/docs.css:122]: Referencing #30, could we auto-scroll the menu to the active item?

**Review by alexroan — CHANGES_REQUESTED** [review]

File: _src/docs.css (hunk 1 above)
Comment (dmitrydao) [reply _src/docs.css:122]: Can you clarify?

Comment (StephenFluin) [reply _src/docs.css:122]: @alexroan Can we land this as a separate feature / PR?

**Review by alexroan — APPROVED** [review]

**Pull request merged**